OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -630,6 +630,23 @@ public static DreamValue NativeProc_clamp(DreamObject instance, DreamObject usr,
             return new DreamValue(0);
         }
 
+        [DreamProc("ftime")]
+        [DreamProcParameter("File", Type = DreamValueType.String)]
+        [DreamProcParameter("IsCreationTime", Type = DreamValueType.Float)]
+        public static DreamValue NativeProc_ftime(DreamObject instance, DreamObject usr, DreamProcArguments arguments) {
```

Comment:
This is missing a `SetGlobalNativeProc()` call in [DreamProcNative.cs](https://github.com/OpenDreamProject/OpenDream/blob/b36a2b68c62c90e86b53bc4e3ffc184d6996dc76/OpenDreamRuntime/Procs/Native/DreamProcNative.cs). Without this it will run the proc defined in DM, which is just an empty proc.